### ISSUE TITLE
Adding postCmd on another container for Watch goal

### ISSUE THREAD

**Issue by sebglon** [rhuss/docker-maven-plugin#370]
Is it possible to add postCmd on another Container B when `watch` show changes on container A?

**Comment by rhuss** [comment]
Sorry, thats currently not possible. What is your use case ?

**Comment by sebglon** [comment]
I change my dataContainer than is linked to tomcatContainer with volume-from.
Is it possible to run cmd on tomcat container when dataContainer is updated?

**Comment by rhuss** [comment]
I assume you use the  `copy` watch mode, otherwise linking doesn't work nicely anyway. I guess you need something additional because normally Tomcat should do a redeploy when the war file is copied into the data container. Is this correct ?

**Comment by sebglon** [comment]
yes but this scenario is available on another container (mysql, apache, ...)
